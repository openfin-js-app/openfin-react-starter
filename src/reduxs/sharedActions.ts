import {
    APPLICATION_SET_LOADING_MSG,
    APPLICATION_LAUNCH_NEW_WINDOW,
} from './application/actions';

import {
    CLIENT_SET_VALUE,
} from './client/actions';

const sharedActions:string[]=[
    APPLICATION_SET_LOADING_MSG,
    APPLICATION_LAUNCH_NEW_WINDOW,
    CLIENT_SET_VALUE,
];

export default sharedActions;