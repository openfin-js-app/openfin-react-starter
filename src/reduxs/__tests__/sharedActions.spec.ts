import sharedActions from '../sharedActions';

describe('shared actions',()=>{

    it('all shared actions', ()=>{
        expect(sharedActions).toMatchSnapshot();
    })

})
