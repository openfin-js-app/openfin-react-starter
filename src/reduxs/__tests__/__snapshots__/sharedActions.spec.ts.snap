// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared actions all shared actions 1`] = `
Array [
  "APPLICATION_SET_LOADING_MSG",
  "APPLICATION_LAUNCH_NEW_WINDOW",
  "CLIENT_SET_VALUE",
]
`;
